retrofit/src/main/java/retrofit2/CustomCallback.java

```diff
@@ -0,0 +1,15 @@
+package retrofit2;
+
+import Call;
```

Comment:
These "imports" are unnecessary and break build. Same for other files. Please remove them